### ISSUE TITLE
Some question  I encountered when  training a  content selection model.

### ISSUE THREAD

**Issue by shellycsy** [ucfnlp/summarization-sing-pair-mix#10]
In the Content Selection model, the best checkpoint you can give is 520,000,But when I trained the model, the best checkpoint I got was within 30,000, and my eval_loss and loss were always equal. During the training process, the loss value has been increasing from the beginning of 0.7 to around 3.1.  I feel very strange, but I didn't find out why. Can you tell me why?

**Comment by loganlebanoff** [comment]
That is strange. Do you know if you changed any of the code before training? And are you training on CNN/DM?

**Comment by shellycsy** [comment]
Yes，I trained on the cnn/dm dataset.&nbsp;Before my training, I removed the blank lines in the original data and changed no other code.




------------------ Original ------------------

**Comment by loganlebanoff** [comment]
Ok, can you post the output of the training command? This one:

python run_classifier.py --task_name=merge --do_train --do_eval --dataset_name=cnn_dm --num_train_epochs=1000.0

**Comment by shellycsy** [comment]
I used the command you provided, unchanged






------------------ Original ------------------

**Comment by loganlebanoff** [comment]
Can you post the output of that command?

**Comment by shellycsy** [comment]
> Can you post the output of that command?

INFO:tensorflow:Restoring parameters from /home/csy/summarization-sing-pair-mix/data/bert/cnn_dm/both/output/model.ckpt-184000
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
2019-11-22 13:27:30.198824: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:98] Filling up shuffle buffer (this may take a while): 78001 of 106765
2019-11-22 13:27:32.581560: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:136] Shuffle buffer filled.
INFO:tensorflow:Evaluation [10/100]
INFO:tensorflow:Evaluation [20/100]
INFO:tensorflow:Evaluation [30/100]
INFO:tensorflow:Evaluation [40/100]
INFO:tensorflow:Evaluation [50/100]
INFO:tensorflow:Evaluation [60/100]
INFO:tensorflow:Evaluation [70/100]
INFO:tensorflow:Evaluation [80/100]
INFO:tensorflow:Evaluation [90/100]
INFO:tensorflow:Evaluation [100/100]
INFO:tensorflow:Finished evaluation at 2019-11-22-05:27:34
INFO:tensorflow:Saving dict for global step 184000: eval_accuracy = 0.545, eval_loss = 2.9479458, global_step = 184000, loss = 2.9479458
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 184000:
